presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/StreamPreferredProperties.java

```diff
@@ -96,6 +96,14 @@ public StreamPreferredProperties withParallelism()
         return new StreamPreferredProperties(Optional.of(MULTIPLE), Optional.empty(), orderSensitive);
     }
 
+    public StreamPreferredProperties withFixedParallelism()
```

Comment:
`Ensure constant number of LookupJoinOperators ` looks good but I am not an expert in `Add*Exchanges` area.